pkg/manager/aws.go

```diff
@@ -143,10 +146,11 @@ func (m *AWS) runComplianceCheck(ctx context.Context) error {
 	if currentCheckoutInfo.EntitledLicenses == requiredLicenses {
 		statusMessage = "Rancher server has the required amount of licenses"
 	} else {
-		statusMessage = fmt.Sprintf("server is not in compliance, wanted %d, but got %d", requiredLicenses, currentCheckoutInfo.EntitledLicenses)
+		statusMessage = fmt.Sprintf("Server is not in compliance with license terms. At least %d more licens(es) are required in %s to become compliant.", requiredLicenses-currentCheckoutInfo.EntitledLicenses, awsCSP)
 	}
+	configMessage := fmt.Sprintf("Rancher server required %d licens(es) and was able to check out %d licens(es)", requiredLicenses, currentCheckoutInfo.EntitledLicenses)
```

Comment:
can you comment here or in updateAdapterOutput that this goes into the support config (if it doesn't already)